content/documentation/getting-started/index.adoc

```diff
@@ -63,51 +66,88 @@ If the Kiali version packaged with Istio or Maistra meets your needs then instal
 
 If you prefer to use the latest Kiali version then do not install Kiali during the Istio or Maistra installation procedure. Complete the Istio or Maistra installation and then install Kiali using the steps below. Note that the Kiali installation script documented below will ask to delete an existing Kiali installation, if found.
 
-=== Install the Kiali Operator
 
-The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
+=== Install Kiali Operator
+
+If you did not install Kiali with Istio or Maistra, or want to replace the installed version of Kiali, it is done using the Kiali Operator. The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
+
+All of the installation options below:
 
-To install the Kiali Operator and, by default, the Kiali CR (custom Resource), run the install script via this command:
+* Ask to delete an existing Kiali Operator, or Kiali installation, when appropriate.
+* Prompt for the authentication strategy to use and/or additional information needed for login.
+** See link:#_login_options[Login Options] for more on the `login`, `anonymous` and `openshift` authentication strategies.
+
```

Comment:
the above is confusing. It doesn't flow. Not sure what its trying to convey.

Reply:
I will replace the bullets with prose and see if you like it better.